### ISSUE TITLE
`test_std::archive_net` failing on `mips-unknown-linux-gnu`

### ISSUE THREAD

**Issue by djkoloski** [rkyv/rkyv#316]
Surfaced in [a recent job](https://github.com/rkyv/rkyv/actions/runs/3192861765/jobs/5210926280). There was a successful run in [the job immediately prior](https://github.com/rkyv/rkyv/actions/runs/3029309879/jobs/4874671224) so this problem appears to have been introduced in the last 25 days. It is reproducible locally with:

```
cross test --package rkyv_test --target mips-unknown-linux-gnu --verbose
```

**Comment by djkoloski** [comment]
Repro minimized upstream at https://github.com/rust-lang/rust/issues/102722

**Comment by djkoloski** [comment]
With some help this has been determined to be an issue with `cross`, now reported at https://github.com/cross-rs/cross/issues/1060. Downgrading to the last published version of `cross` resolves this issue, which has been done in #317.
